util/src/test/java/io/camunda/zeebe/util/buffer/BufferUtilTest.java

```diff
@@ -111,6 +112,34 @@ public void shouldNotReturnWrappedArrayIfBufferOffsetIsZero() {
     assertThat(bytes).isNotSameAs(BYTES1);
   }
 
+  @Test
+  public void shouldTrimBufferIfExceeds() {
+    // given
+    final int maxLength = BYTES1.length - 1;
+    final byte[] expected = new byte[maxLength];
+    System.arraycopy(BYTES1, 0, expected, 0, maxLength);
+    final DirectBuffer expectedBuffer = new UnsafeBuffer(expected);
+    final DirectBuffer src = new UnsafeBuffer(BYTES1);
+
+    // when
+    final DirectBuffer dst = trimBufferIfExceeds(src, maxLength);
```

Comment:
❓ I'm just wondering what `dst` stands for

Reply:
@remcowesterhoud it means `destination`. I used it here since this name is also used in common Java libraries (e.g. `String`), so sounds familiar. But feel free to suggest a new one if you think it will improve readability.